### PR TITLE
MVP-1: styling for file upload error alert

### PR DESCRIPTION
**Pull request by KatiRG**
## What this PR accomplishes

Changes the style of the error alert box on Step 1 of the new workflow to be consistent with the DS.

## What needs review
note: review with https://github.com/ongov/ckanext-validation/pull/3

Upload a file that contains invalid headers (e.g. duplicate headers or header names > 63 characters) and confirm that:

 - alert box styling is consistent with DS
 - the hyperlinked "validation report" opens the validation report modal, as before

Also check that the entire workflow works as expected once a fixed file has been uploaded.

### CODE REVIEW

File: ckanext/ontario_theme/templates/internal/scheming/snippets/errors.html [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+                        {%- endif -%}
+                      {%- endfor -%}
+                    </ul>
+                  </li>
+                {%- endif -%}
+              {%- endfor -%}
+            {%- else -%}
+              {# Missing mandatory fields #}
+              <li data-field-label="{{ field.field_name }}">{{
+                h.scheming_language_text(field.label) }}:
```
Comment (aleeexgreeen) [inline ckanext/ontario_theme/templates/internal/scheming/snippets/errors.html:74]:
```suggestion
              <li data-field-label="{{ field.field_name }}">{{ h.scheming_language_text(field.label) }}:
```

File: ckanext/ontario_theme/templates/internal/scheming/snippets/errors.html [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+                          {%- endif -%}
+
+                          {%- snippet sfe_snippet, unprocessed=se_unprocessed,
+                            field=sf, fields=field.repeating_subfileds,
+                            entity_type=entity_type, object_type=object_type -%}
+                        {%- endif -%}
+
+                        {%- if sf.field_name in se_unprocessed -%}
+                          <li data-field-label="{{ field.field_name }}-{{ loop.index }}-{{ sf.field_name }}">{{
+                            h.scheming_language_text(sf.label) }}:
```
Comment (aleeexgreeen) [inline ckanext/ontario_theme/templates/internal/scheming/snippets/errors.html:63]:
```suggestion
                          <li data-field-label="{{ field.field_name }}-{{ loop.index }}-{{ sf.field_name }}">{{ h.scheming_language_text(sf.label) }}:
```

File: ckanext/ontario_theme/plugin.py [hunk 3]
```diff
@@ -1108,7 +1108,8 @@ def get_helpers(self):
                 'ontario_theme_get_facet_options': get_facet_options,
                 'ontario_theme_site_title': site_title,
                 'ontario_theme_get_current_year': get_current_year,
-                'ontario_theme_get_validation_report': get_validation_report
+                'ontario_theme_get_validation_report': get_validation_report,
+                'validation_extract_report_from_errors': validation_extract_report_from_errors
```
Comment (aleeexgreeen) [inline ckanext/ontario_theme/plugin.py:1112]:
```suggestion
```
Turns out this isn't needed after all!

File: ckanext/ontario_theme/templates/internal/scheming/snippets/errors.html
Comment (aleeexgreeen) [inline ckanext/ontario_theme/templates/internal/scheming/snippets/errors.html:1]: In your virtual environment, can you run `pre-commit install`. And then with your staged changes of this file, can you run `pre-commit run --hook-stage manual format`. You'll need to review the formatting changes and restage the file.

File: ckanext/ontario_theme/templates/internal/scheming/snippets/errors.html [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+    {% if validation_report %}
+      {% if h.bootstrap_version() == '3' %}
+        {% snippet 'validation/snippets/validation_report_dialog.html', validation_report=h.dump_json_value(validation_report) %}
+      {% else %}
+        {% snippet 'validation/snippets/validation_report_dialog_bs2.html', validation_report=h.dump_json_value(validation_report) %}
+      {% endif %}
+    {% endif %}
+  </div>
+
+{% endblock %}
```
Comment (aleeexgreeen) [inline ckanext/ontario_theme/templates/internal/scheming/snippets/errors.html:99]:
```suggestion
{% endblock errors_list%}
```

File: ckanext/ontario_theme/templates/internal/scheming/snippets/errors.html [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+        {%- endfor -%}
+
+        {%- for key, errors in unprocessed.items() | sort -%}
+          {% if key == "validation" %}
+            <li data-field-label="{{ key }}">{{ errors[0] }}</li>
+          {% else %}
+            <li data-field-label="{{ key }}">{{ _(key) }}: {{ errors[0] }}</li>
+          {% endif %}
+        {%- endfor -%}
+      {% endblock %}
```
Comment (aleeexgreeen) [inline ckanext/ontario_theme/templates/internal/scheming/snippets/errors.html:87]:
```suggestion
      {% endblock all_errors %}
```

**Review by aleeexgreeen — CHANGES_REQUESTED** [review]

File: ckanext/ontario_theme/templates/internal/scheming/snippets/errors.html
Comment (KatiRG) [reply ckanext/ontario_theme/templates/internal/scheming/snippets/errors.html:1]: I cannot get it to run. Do you need more fomatting to merge this PR?

Comment (KatiRG) [reply ckanext/ontario_theme/templates/internal/scheming/snippets/errors.html:1]: I checked manually and the lint and formatting should be fine

**Review by aleeexgreeen — APPROVED** [review]

**Pull request merged**